salt/renderers/dson.py

```diff
@@ -20,14 +20,18 @@
 try:
     import dson
 except ImportError:
-    raise
+    dson = None
 
 # Import salt libs
 from salt.ext import six
 
 log = logging.getLogger(__name__)
 
 
+def __virtual__():
+    return dson, 'Failed to load: dson module not installed'
```

Comment:
This will probably break if `dson` _is_ installed. The first value in the tuple should be either a string or a bool, in this case it's either a module or `None`.